pkg/blockstorage/azure/azuredisk.go

```diff
@@ -722,3 +701,15 @@ func (s *AdStorage) dynamicRegionMapAzure(ctx context.Context) (map[string][]str
 	}
 	return regionMapResult, nil
 }
+
+func (s *AdStorage) mapLocationToZone(skuResult *armcompute.ResourceSKU, regionMap *map[string]LocationZoneMap) {
+	var rm = *regionMap
+	for _, locationInfo := range skuResult.LocationInfo {
+		if val, ok := rm[*locationInfo.Location]; ok {
```

Comment:
[nil pointer] ditto on guard for `nil` pointer

Reply:
Done